### DISCUSSION TITLE
Chapter mark on quote and dedicatory

### DISCUSSION THREAD

**Discussion by saraa-costa** [joaomlourenco/novathesis#462] [Q&A] [answered]
Hi,
I'm using miktex and TeXworks. When I compile, the chapter mark appears on quote and dedicatory, as shown below.
![Captura de ecrã 2023-09-17 093716](https://github.com/joaomlourenco/novathesis/assets/131374649/46fb9393-e9df-4a89-9ecf-e24d057de0db)

**Comment by joaomlourenco** [comment]
Hi. Thanks for reporting…. you're using an unusual chapter style that was not properly tested.
Can you tell me which version of the template are you using?
Thanks.

**Reply by saraa-costa** [reply]
Hi,
I don’t know where to see that but I do the download of zip file four days ago and I start to use Miktex. In overleaf I don’t see this bug

**Comment by joaomlourenco** [comment]
The version of the template is near the top of the files "template.tex".  Open the file and find the version number.

**Comment by saraa-costa** [comment]
I'll send you a print so there's no mistake, but I think it's the version "Version 2023-08-14 [7.1.5]"
![image](https://github.com/joaomlourenco/novathesis/assets/131374649/846989a0-d8fd-47b5-84f8-6165e3aecfb7)

**Comment by joaomlourenco** [comment, marked as answer]
Fixed it. :)
Please replace your file “novathesis.cls” with this one:
https://raw.githubusercontent.com/joaomlourenco/novathesis/develop/novathesis.cls

Let me know if it worked for you.

**Reply by saraa-costa** [reply]
It worked :)
Thank you
